index.css

```diff
@@ -0,0 +1,137 @@
+* {
+    box-sizing: border-box;
+    background-size: auto auto;
```

Comment:
'background-size' property specifies the size of a background-image. But we are not using any background image here.